### ISSUE TITLE
France- Missing food and paper sectors in the front-end but non-zero values in the back-end

### ISSUE THREAD

**Issue by nairnamratha** [quintel/etmodel#2534]
When the France dataset is used in the ETM model, the sub-sectors "Food" and "Paper" are not shown in the front end. 
![screen shot 2017-11-08 at 14 24 47](https://user-images.githubusercontent.com/32448293/32551394-ab56c572-c490-11e7-8fb2-e423cefd370a.png)

However, the energy graph shows these sectors to have useful demand values which are non-zero. 

![screen shot 2017-11-08 at 14 25 04](https://user-images.githubusercontent.com/32448293/32551397-ae2a12e0-c490-11e7-86a8-1e5e2b9e1991.png)

How is this so?

**Comment by antw** [comment]
Those sectors are only shown when the region `has_detailed_other_industry ` is set to `true`. [This is not the case for France](https://github.com/quintel/etsource/blob/master/datasets/fr/fr.full.ad#L25-L28) which uses a simplified version of those sectors.

I don't know if it's correct for those nodes to have a demand when the detailed model is turned off, but I *assume* that it is. @DorinevanderVlies: do you know?

**Comment by nairnamratha** [comment]
Thanks @antw ! Hadn't noticed the has_detailed_other_industry earlier.

**Comment by DorinevanderVlies** [comment]
> I don't know if it's correct for those nodes to have a demand when the detailed model is turned off, but I assume that it is. @DorinevanderVlies: do you know?

I am not sure either. I assume this decision was made for all dataset when industry was split in different sectors. Is that correct @jorisberkhout 

I would propose to set `has_detailed_other_industry` to `true` for all datasets in which this detailed industry has a value. So for France that would imply that this setting is changed. What do you think @jorisberkhout?

**Comment by RobTerwel** [comment]
> Is that correct @jorisberkhout

Yes
> I would propose to set has_detailed_other_industry to true for all datasets in which this detailed industry has a value.

This doesn't work without some additional research. Let me try to clarify 😄 

If `has_detailed_other_industry` is set to `false`, all electricity consumption of the paper or food sector (as follows straight from the EB) is assigned to the electricity UD of that sector. If `has_detailed_other_industry` is set to `true`, however, one has to specify *what part of the electricity use of the subsector is used for heat generation*. That then determines the share of the FD of electricity of the subsector that goes to the UD and the share that goes to the electric heater. This is done in the `industry_analysis`.

In short, everything is fine as it is!

**Comment by DorinevanderVlies** [comment]
Thanks for the clarification @jorisberkhout and @RobTerwel
